src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml

```diff
@@ -46,20 +31,6 @@
 			class="edu.cornell.kfs.sys.document.validation.impl.AccountingLineRequiredOverridesValidation" abstract="true">
 		<property name="accountingLineRuleHelperService" ref="purchaseOrderAmendmentAccountingLineRuleHelperService"/>
 	</bean>
-
-	<bean id="Requisition-newIndividualItemValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.RequisitionNewIndividualItemValidation" abstract="true">  
```

Comment:
The custom cu-kfs version of RequisitionNewIndividualItemValidation does not extend from the one in base code, and it does not have the commodityCodeIsRequired() method override like in the base code's class. Can you please confirm if we should be using the base code's variant instead? If so, then can we remove the cu-kfs version of RequisitionNewIndividualItemValidation?

Reply:
Restoring entry for RequisitionNewIndividualItemValidation